### PR TITLE
Feat: RTENU-126 page contents updating

### PR DESCRIPTION
**Pull request by mehtis**
* Added writeRights validation support
* Added db fields update to test writeRights check
* Added static admin role
* Added http methods to alb
* Refactored lambda policy addition to avoid repetition
* Removed unnecessary awaits from validation calls
* Updated sam:invoke instructions, as AWS profile is needed now for some reason
* Added support for using local db with sam
* Updated getRouterNames to remove even more special characters
* Hack fix for api 403 errors redirecting to /index.html, using 401 instead

### CODE REVIEW

File: packages/server/utils/userService.ts [hunk 1]
```diff
@@ -82,25 +83,27 @@ export const getUser = async (event: ALBEvent): Promise<RataExtraUser> => {
...
-      throw new RataExtraLambdaError('Forbidden', 403);
-    }
+/**
+ * Checks if the user has necessary role for write access. Also checks for admin rights. Throws 403 Forbidden if not authorised.
+ * @param {RataExtraUser} user User being validated
+ * @param {string} writeRole Role being validated against
+ */
+export const validateWriteUser = (user: RataExtraUser, writeRole: string): void => {
+  if (isAdmin(user)) {
+    return;
```
Comment (NinaDang97) [inline packages/server/utils/userService.ts:102]: Or ?
```
if (isAdmin(user) || isWriteUser(user, writeRole)) {
    return;
  } else {
log.error(user, 'Forbidden: User is not an authorised write user');
    throw new RataExtraLambdaError('Forbidden', 403);
}
```

File: packages/server/lambdas/database/client/index.ts [hunk 2]
```diff
@@ -11,15 +13,21 @@ let databasePassword: string | null = null;
...
   public static async build(): Promise<PrismaClient> {
-    if (!databaseName || !databaseDomain || !databasePassword) {
-      [databaseName, databaseDomain, databasePassword] = await Promise.all([
-        getParameter(databaseNameId),
-        getParameter(databaseDomainId),
-        getSecuredStringParameter(databasePasswordId),
-      ]);
-    }
+    let DATABASE_URL: string;
+    const setLocalDatabaseConnection = environment === ENVIRONMENTS.local;
```
Comment (NinaDang97) [inline packages/server/lambdas/database/client/index.ts:17]: should be `isLocalDatabaseConnection` ?

File: packages/server/utils/userService.ts (hunk 1 above)
Comment (mehtis) [reply packages/server/utils/userService.ts]: Good call. Updated

File: packages/server/lambdas/database/client/index.ts (hunk 2 above)
Comment (mehtis) [reply packages/server/lambdas/database/client/index.ts]: Wasn't happy with set, but didn't go for is for some reason. Fixed!

File: packages/server/lambdas/database/client/index.ts [hunk 3]
```diff
@@ -11,15 +13,21 @@ let databasePassword: string | null = null;
...
+    if (setLocalDatabaseConnection) {
+      DATABASE_URL = 'postgresql://root:root@host.docker.internal:5432/test_db?schema=public';
+    } else {
```
Comment (NinaDang97) [inline packages/server/lambdas/database/client/index.ts:20]: We are not allowed to get ssm keys if locally? 
How about remove if-else for `setLocalDatabaseConnection` and replace with this?
``` 
let DATABASE_URL = setLocalDatabaseConnection ? "postgresql://root:root@host.docker.internal:5432/test_db?schema=public" : "postgresql://${databaseName}:${databasePassword}@${databaseDomain}:5432/${databaseName}?schema=public"
```

Comment (mehtis) [reply packages/server/lambdas/database/client/index.ts:20]: If we're running locally, there's no need for SSM stuff.

**Review by NinaDang97 — APPROVED** [review]

**Pull request merged**